app/utils/latex.ts

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+/**
+ * Splits up latex code in an array consisting of each line of latex code.
+ * A latex linebreak is denoted by `\\`.
+ * A line is not broken in 2 cases:
+ * - The linebreak is in between brackets.
+ * - The linebreak occurs inbetween a `\begin` and `\end` statement.
+ */
+export function splitLatexCodeInLines(content: string): Array<string> {
```

Comment:
The linebreaks inside of a begin/end statement or curly braces are handled correctely by the `react-native-math-view` library.

For example when creating a matrix:
```
\begin{matrix}
1 & 2 & 3\\
a & b & c
\end{matrix}
```

The matrix still needs to be printed on 1 line because the linebreaks appear inside of the matrix statement. 

The same thing happens when a line break is in between curly braces. An example of this is the following:
```
\prod_{\substack{1\le i \le n\\ 1\le j \le m}}M_{i,j}
```

Reply:
Ah, I see. So that's needed because the react-native-math-view doesn't know how to handle the line breaks itself when it would better understand how to apply them in different contexts.

Thanks for the examples since I've only got a very limited understanding of Latex.